pyclean/modern.py

```diff
@@ -400,3 +405,95 @@ def suggest_debris_option(args):
             'Hint: Use --debris to also clean up build artifacts '
             'from common Python development tools.',
         )
+
+
+def build_git_clean_command(ignore_patterns, dry_run=False, force=False):
+    """
+    Build the git clean command with appropriate flags.
+
+    Args:
+        ignore_patterns: List of patterns to exclude from git clean
+        dry_run: If True, only show what would be deleted
+        force: If True, delete without prompting (otherwise interactive)
+
+    Returns:
+        List of command arguments
```

Comment:
Remove this verbose docstring body. Use type annotations instead if supported by the lowest Python version we cover.